### PR TITLE
Reorg test code

### PR DESCRIPTION
**Pull request by platfowner**
Change summary:
- Reorg test code

Related issues:
- https://github.com/ainblockchain/ain-blockchain/issues/762

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -17,35 +17,35 @@
...
+    "test_integration_consensus": "./node_modules/mocha/bin/mocha --timeout 640000 test/integration/consensus.test.js",
+    "test_integration_function": "./node_modules/mocha/bin/mocha --timeout 640000 test/integration/function.test.js",
+    "test_integration_node": "./node_modules/mocha/bin/mocha --timeout 640000 test/integration/node.test.js",
+    "test_integration_he_protocol": "./node_modules/mocha/bin/mocha --timeout 640000 test/integration/he_protocol.test.js",
+    "test_integration_he_sharding": "./node_modules/mocha/bin/mocha --timeout 640000 test/integration/he_sharding.test.js",
+    "test_integration_sharding": "./node_modules/mocha/bin/mocha --timeout 640000 test/integration/sharding.test.js",
+    "test_unit": "ACCOUNT_INDEX=0 PEER_CANDIDATE_JSON_RPC_URL='' BLOCKCHAIN_CONFIGS_DIR=blockchain-configs/1-node ./node_modules/mocha/bin/mocha --timeout 160000 \"test/unit/*.test.js\"",
     "test_unit_block_pool": "ACCOUNT_INDEX=0 PEER_CANDIDATE_JSON_RPC_URL='' BLOCKCHAIN_CONFIGS_DIR=blockchain-configs/1-node ./node_modules/mocha/bin/mocha --timeout 160000 unittest/block-pool.test.js",
```
Comment (liayoo) [inline package.json:30]: should these be updated as well? (L30-33)

Comment (platfowner) [reply package.json]: Oh, right. Fixed and tested individually. Thanks!

**Comment by platfowner** [comment]
All comments addressed. PTAL~ @liayoo

**Review by liayoo — APPROVED** [review]
LGTM, thanks!

**Pull request merged**